### ISSUE TITLE
The Last Line of Defence

### ISSUE THREAD

**Issue by pegesus328** [dalaranwow/dalaran-wow#5390]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: On the quest, the Last Line of Defence, at the Argent Vanguard in Icecrown, whilst manning the turrets at the top of the tower I am unable to kill the Frostbrood Destroyers. They simply do not take any damage from the rockets, and when you cast The Reckoning Bomb, it does some damage, but then they go back to their spawn points and gain full health. The Scourge Attackers also evade many of the attacks, however for a period they do take damage and can be killed. 

**Expected behaviour**: The Frost Destroyers should take damage from the rockets and The Reckoning Bomb. 

**Steps to reproduce the problem**:

1. Take the quest the Last Line of Defence
2. Use one of the Argent Cannons
3. Attack the Frostbrood Destroyer with either the Fire or the Reckoning Bomb

**Include proofs for this behaviour**
http://www.wowhead.com/quest=13086/the-last-line-of-defense
https://www.youtube.com/watch?v=8sD5UN067m8

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Argent Cannon
Frostbrood Destroyer
Scourge Attacker

**Include Screenshots from the issue if necessary**

**Comment by Nicadimos** [comment]
I can second this. Though I think its an issue with most vehicle based quests. Had the same issue with the Mammoth riding quest in ZulDrak

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/5386
